R/cec.R

```diff
@@ -0,0 +1,73 @@
+#' Calculate the index for the CEC-occupation
+#' 
+#' This function calculates the CEC-buffer index for a soil
+#' 
+#' @param A_CEC_CO (numeric) The cation exchange capacity (mmol+ / kg) 
+#' @param A_K_CEC (numeric) The occupation of the CEC with K (\%)
+#' @param A_CA_CEC (numeric) The occupation of the CEC with Mg (\%)
+#' @param A_MG_CEC (numeric) The occupation of the CEC with Ca (\%)
+#' @param advice (character) Optional parameter to select CEC index for soil structure or fertility. Options: fertility_index or structure_index
+#'    
+#' @import data.table
+#' 
+#' @export
+calc_cec <- function(A_CEC_CO,A_K_CEC,A_CA_CEC,A_MG_CEC, advice ='fertility_index') {
```

Comment:
Is default necessary?

Reply:
not necessarily. might indeed be better that users explicietly give this argument since it affects the meaning of the calculated index.